junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/JupiterTestDescriptor.java

```diff
@@ -103,6 +110,46 @@
 
 	// --- Node ----------------------------------------------------------------
 
+	protected ExecutionMode getExecutionMode(AnnotatedElement element) {
+		// @formatter:off
+		return findAnnotation(element, Execution.class)
+				.map(Execution::value)
+				.map(JupiterTestDescriptor::toExecutionMode)
+				.orElseGet(() -> getParent()
+						.filter(parent -> parent instanceof Node)
+						.map(parent -> ((Node<?>) parent).getExecutionMode())
+						.orElse(ExecutionMode.CONCURRENT));
+		// @formatter:on
```

Comment:
Will Jupiter now default to concurrent execution?

Reply:
Only if the `junit.jupiter.execution.parallel.enabled` config param is set tot `true`.

Reply:
OK. In that code it looks like it's defaulting to concurrent, so I'll have to review in greater depth to find where the flag is honored.

Reply:
Just have a look at `JupiterTestEngine`. 😉

Reply:
Alrighty. Will do...